portfolio/src/main/webapp/script.js

```diff
@@ -195,13 +261,81 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+/**
+ * Disables some of the pagination buttons if the current page
+ * is the first or the last one.
+ */
+function disablePaginationButtons() {
+  let disable = false;
+  if (getCurrentPage() === 1) {
+    // disables first and previous page button
+    disable = true;
+  }
```

Comment:
You can make this section shorter: `const disable = getCurrentPage() === 1;`

Reply:
Probably not a `const` because I reassign it.